config/crd/bases/oadp.openshift.io_nonadminbackups.yaml

```diff
@@ -619,6 +619,19 @@ spec:
                 - Created
                 - Deleting
                 type: string
+              queueInfo:
+                description: |-
+                  QueueInfo holds the queue position for a specific VeleroBackup.
+                  It is used to estimate how many backups are scheduled before the given VeleroBackup in the OADP namespace.
+                  This number is not guaranteed to be accurate, but it should be close. It's inaccurate for cases when
+                  Velero pod is not running or being restarted after Backup object were created.
+                  It counts only VeleroBackups that are still subject to be handled by OADP/Velero.
+                properties:
+                  estimatedQueuePosition:
+                    type: integer
+                required:
```

Comment:
why it is required?

Reply:
To be consistent across all NAB objects. Our controller does the creation and handling so it's up to us.